Sources/MapboxMaps/Foundation/Extensions/Core/ResourceOptions.swift

```diff
@@ -3,22 +3,30 @@ import Foundation
 // MARK: - ResourceOptions
 
 extension ResourceOptions {
-    /// Intializer for `ResourceOptions`
+
+    /// Initialize a `ResourceOptions`, used by both `MapView`s and `Snapshotter`s
     /// - Parameters:
-    ///   - accessToken: Your mapbox access API token.
-    ///   - baseUrl: The base URL.
-    ///   - cachePath: The path to the cache.
-    ///   - assetPath: The path to the assets.
-    ///   - tileStorePath: The path to the tile store.
-    ///   - loadTilePacksFromNetwork: Detemines if tiles pack should or should not be avoided from the network.
-    ///   - cacheSize: The size of the cache.
+    ///   - accessToken: Mapbox access token. You must provide a valid token.
+    ///   - baseUrl: Base url for resource requests; default is `nil`
+    ///   - cachePath: Path to database cache; default is `nil`, which will create
+    ///         a path in the application's "support directory"
+    ///   - assetPath: Path to assets; default is `nil`, which will use the application's
+    ///         resource bundle.
+    ///   - tileStorePath: Path to the `TileStore`; if `nil` the Tile Store will
+    ///         not be used.
+    ///   - loadTilePacksFromNetwork: Enables or disables tile store packs loading
+    ///         from network; default is `true`. This value will be ignored if
+    ///         the `tileStorePath` is `nil`.
+    ///   - cacheSize: Size of the cache.
     public convenience init(accessToken: String,
                             baseUrl: String? = nil,
                             cachePath: String? = nil,
                             assetPath: String? = nil,
                             tileStorePath: String? = nil,
-                            loadTilePacksFromNetwork: NSNumber? = nil,
+                            loadTilePacksFromNetwork: Bool = true,
```

Comment:
can you mark this as done on the public API review doc we have

Reply:
Let's leave that as a reminder - since it really needs `NS_REFINED_FOR_SWIFT`.

Reply:
These are managed via peer objects.